whiz.yaml

```diff
@@ -19,7 +19,7 @@ gate1: &tp
     REDIS_URL: "redis://:password@localhost:6379/0"
     TG_SECRET: "a4lNi0PbEItlFZbus1oeH/+wyIxi9uH6TpL8AIqIaMBNvp7SESmuUBbfUwC0prxhGhZqHw8vMDYZAGMhSZ4fLw=="
     TG_ADMIN_PASSWORD: "password"
-    LD_LIBRARY_PATH: /data/home/metatype/.ghjk/envs/default/shims/lib
```

Comment:
is that wanted?

Reply:
The path absolute. It was specific to his setup when he added it there...

Reply:
Yeah, should it be `./.ghjk/envs/default/shims/lib` instead?

Reply:
Yeah, this is no longer necessary since #754 .